views/partials/component/container.html

```diff
@@ -0,0 +1,29 @@
+<div class="o-registry-ui__component--container">
+
+
+	{{> component/header}}
+
+	{{#if demos}}
+		{{> component/type/demos}}
+	{{/if}}
+
+	{{#if images}}
+		{{> component/type/images}}
+	{{/if}}
+
+	{{#if service}}
+		{{> component/type/service}}
+	{{/if}}
+
+	<div class="o-registry-ui__component--support">
+		<div class="o-registry-ui__component--contact">
+		<h4>Support</h4>
+			<p><span>EMAIL </span><a href="mailto:{{component.support.email}}?subject={{component.name}}">{{component.support.email}}</a></p>
+			{{#if component.support.channel}}
+			<p><span>SLACK </span><a href="{{component.support.channel.url}}">{{component.support.channel.name}}</a></p>
+			{{/if}}
+		</div>
+		<a href="{{component.url/issue}}" class="secondary-button">REPORT AN ISSUE ON GITHUB</a>
```

Comment:
This can be taken from the JSON 🙂 repo data puts all the URLs together for you
```diff
-{{component.url/issue}}
+{{component.supportingUrls.issues}}
```